routemaster/models/event.rb

```diff
@@ -23,6 +23,7 @@ def initialize(**options)
         @data      = EventData.build options[:data]
 
         _assert VALID_TYPES.include?(@type), 'bad event type'
+        _assert options[:timestamp].to_i <= Routemaster.now, 'timestamp is from the future'
```

Comment:
see i'm late to this party, but seems pretty harsh on clock skew (and leap second handling).  any reason why not a couple second grace?